var/spack/repos/builtin/packages/glib/no-Werror=format-security.patch

```diff
@@ -0,0 +1,16 @@
+--- a/configure.ac	2016-08-16 11:57:34.000000000 -0400
```

Comment:
A lot of websites don't recommend it:
http://www.mtu.edu/umc/services/digital/writing/characters-avoid/
I don't know of any platforms that explicitly forbid it, but it also doesn't add anything.